project/plugins.sbt

```diff
@@ -29,10 +29,10 @@ addSbtPlugin("uk.gov.hmrc" % "sbt-artifactory" % "0.13.0")
 
 addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "1.1.0")
 
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.5.12")
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.5.19")
 
 addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.5.0")
 
 addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "0.8.0")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-bobby" % "0.32.0")
\ No newline at end of file
+addSbtPlugin("uk.gov.hmrc" % "sbt-bobby" % "0.32.0")
```

Comment:
Could you also update:
`mockito-core` -> 2.23.4
`scalatest` -> 3.0.5
`jsoup` -> 1.11.3
`wiremock` -> 2.20.0